collect_stata/__main__.py

```diff
@@ -0,0 +1,25 @@
+"""__main__.py"""
+__author__ = "Marius Pahl"
+
+import argparse
+import sys
+
+from collect_stata.stata_to_json import stata_to_json
+
+
+def main(args=None):
+    """The main routine."""
+    if args is None:
+        args = sys.argv[1:]
+        print(args)
+
+    parser = argparse.ArgumentParser(description="Get input and output information")
+    parser.add_argument("--i", "--input", help="get input path")
```

Comment:
In #3 it is requested to have `-i` and `-o` as arguments. Not --i and --o.

Reply:
Please follow POSSIX [conventions](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html).